### PR TITLE
Fix used services to conform to style guide

### PR DESCRIPTION
**Pull request by NobleDraconian**
I fixed the various instances of `game.<ServiceName>` to use `game:GetService()`, conforming to the [Roblox style guide](https://roblox.github.io/lua-style-guide/).

### CODE REVIEW

File: PlayerScripts/StarterCharacterScripts/Sound/LocalSound.client.lua [hunk 1]
```diff
@@ -9,6 +9,10 @@
 	This script is optimized to reduce network traffic through minimizing the amount of property replication.
 ]]--
 
+--Roblox Services
+local Workspace=game:GetService("Workspace")
+local Players=game:GetService("Players")
```
Comment (Kampfkarren) [inline PlayerScripts/StarterCharacterScripts/Sound/LocalSound.client.lua]: space needed between name and equals sign

**Comment by Kampfkarren** [comment]
This repo is outdated and doesn't take PRs anymore it seems.

File: PlayerScripts/StarterCharacterScripts/Sound/LocalSound.client.lua (hunk 1 above)
Comment (NobleDraconian) [reply PlayerScripts/StarterCharacterScripts/Sound/LocalSound.client.lua]: Yeah I'll fix that

Comment (NobleDraconian) [reply PlayerScripts/StarterCharacterScripts/Sound/LocalSound.client.lua]: Just fixed it.

File: PlayerScripts/StarterCharacterScripts/Sound/LocalSound.client.lua [hunk 2]
```diff
@@ -9,6 +9,10 @@
 	This script is optimized to reduce network traffic through minimizing the amount of property replication.
 ]]--
 
+--Roblox Services
+local Workspace = game:GetService("Workspace")
+local Players = game:GetService("Players")
```
Comment (Kampfkarren) [inline PlayerScripts/StarterCharacterScripts/Sound/LocalSound.client.lua:13]: needs to be in alpha order

Comment (NobleDraconian) [reply PlayerScripts/StarterCharacterScripts/Sound/LocalSound.client.lua:13]: Updated.

**Comment by cliffchapmanrbx** [comment]
Hi there, thank you for your feedback. We're going to be closing this repository as we've chosed [the Roblox Dev Forum](https://devforum.roblox.com/) as our venue for developer feedback. Please feel free to migrate your feedback to a DevForum post if you feel it is still relevant.

**Pull request closed without merging**